### ISSUE TITLE
Webpage is not compliant

### ISSUE THREAD

**Issue by liamageo** [liamageo/github-issue-template#6]
**Describe the bug**
The webpage has a score of zero, which means that it is at risk of accessibility lawsuits. 

**To Reproduce**
1. Go to 'https://www.accessibilitychecker.org/'
2. Paste in 'https://liamageo.github.io/github-issue-template/mission.html'
4. See errors displayed on the right side of the screen

**Expected behavior**
The website should have a score of 75%+, to avoid the risk of accessibility lawsuits.

**Screenshots**
You can view results by clicking this URL:
https://www.accessibilitychecker.org/audit/?website=https%3A%2F%2Fliamageo.github.io%2Fgithub-issue-template%2Fmission.html&flag=us

**Desktop (please complete the following information):**
 -- Chrome

**Smartphone (please complete the following information):**
 - N/A

**Additional context**
Utilize the recommended providers that the site provides, in order to receive a score of 75% or higher.
